### DISCUSSION TITLE
Winget for Windows Distro?

### DISCUSSION THREAD

**Discussion by MrHinsh** [DistroAV/DistroAV#1065] [Ideas]
Have you considered creating a Winget publish so that folks on Windows can install from the command line?

Winget is the standard package manager for Windows. I'm happy to help set this up! I have it for some of my projects and it can be automated in the workflow...

The initial approval process is a pain, but once in everything else is pretty easy...

**Comment by Trouffman** [comment]
See #1131 
I plan on making the first version available for 6.1
